src/rust/src/x509/certificate.rs

```diff
@@ -1076,6 +1020,74 @@ pub(crate) fn set_bit(vals: &mut [u8], n: usize, set: bool) {
     }
 }
 
+pub(crate) fn identify_signature_algorithm_parameters<'p>(
+    py: pyo3::Python<'p>,
+    params: &common::AlgorithmParameters<'p>,
+    signature_hash_algorithm: &pyo3::PyAny,
+) -> CryptographyResult<&'p pyo3::PyAny> {
+    match params {
+        common::AlgorithmParameters::RsaPss(opt_pss) => {
+            let pss = opt_pss.as_ref().ok_or_else(|| {
+                pyo3::exceptions::PyValueError::new_err("Invalid RSA PSS parameters")
```

Comment:
This is not reachable because this error will already be raised by the `signature_hash_algorithm` call, which is now happens first.